Homework2: Alexandr Anisko

Talion90 opened the pull request:
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

Talion90 committed:
Homework1: Alexandr Anisko

Talion90 committed:
Homework1: Alexandr Anisko

Talion90 committed:
Update 1_hello_world.py

Talion90 committed:
add space and add leading new line

Talion90 committed:
another one

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
another one

Talion90 committed:
another one

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
another

Talion90 committed:
another

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 1_hello_world.py

Talion90 committed:
Update 2_Input.py

Talion90 committed:
Update 2_Input.py

Talion90 committed:
Update 1_hello_world.py

Talion90 committed:
Update 2_Input.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 1_hello_world.py

Talion90 committed:
Update 1_hello_world.py

Talion90 committed:
Update 1_hello_world.py

Talion90 committed:
Update 1_hello_world.py

Talion90 committed:
Update 1_hello_world.py

Talion90 committed:
Update 1_hello_world.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Update 14_doctest.py

Talion90 committed:
Delete 10_time_con.py

Talion90 committed:
Delete 11_triple_q_str.py

Talion90 committed:
Delete 12_classes.py

Talion90 committed:
Delete 13_unit_test.py

Talion90 committed:
Delete 14_doctest.py

Talion90 committed:
Delete 15_itertools.py

Talion90 committed:
Delete 16_csv.py

Talion90 committed:
Delete 17_8-queens_problem_recursion.py

Talion90 committed:
Delete 18_fancy_gen.py

Talion90 committed:
Delete 19_xml_html_parsing.py

Talion90 committed:
Delete 1_hello_world.py

Talion90 committed:
Delete 20_8_queens_problem_exception.py

Talion90 committed:
Delete 21_game_number.py

Talion90 committed:
Delete 2_Input.py

Talion90 committed:
Delete 3_for_loop.py

Talion90 committed:
Delete 4_fibonacci.py

Talion90 committed:
Delete 5_functions.py

Talion90 committed:
Delete 6_import.py

Talion90 committed:
Delete 7_dict_gen_exp.py

Talion90 committed:
Delete 8_cmd_arg.py

Talion90 committed:
Delete 9_open_files.py

Talion90 committed:
Add "№"

Talion90 committed:
Add tasks

Talion90 committed:
Update task2.py

Talion90 committed:
Update task5.py

Talion90 committed:
Update task2.py

Talion90 committed:
5que task from codewars

Talion90 committed:
Add files via upload

5 que task from codewars.com

Talion90 committed:
Add some home tasks from codewars

Talion90 committed:
Merge remote-tracking branch 'origin/Homework№2' into Homework№2

# Conflicts:
#	src/Homework2/task8.py

Talion90 committed:
Update task5.py

Talion90 committed:
Update task8.py

Talion90 committed:
Update task8.py

belovmd commented on src/Homework2/task2.py:
```diff
@@ -0,0 +1,16 @@
+my_string = input("Введите предложение: ").split()
+formatted_string = []
+for word in my_string:
+    if word[len(word) - 1] == ',' or word[len(word) - 1] == '.' \
```
we have ; ( < and many others

belovmd commented on src/Homework2/task5.py:
```diff
@@ -0,0 +1,11 @@
+
+# try to find number with for-loop
+for number in range(number_fib - 1):
+    count = number_first + number_second
```
a, b = b, a+b

belovmd commented on src/Homework2/task6.py:
```diff
@@ -0,0 +1,26 @@
+    ost = my_number % 10
+    rev_number += ost
+    rev_number = rev_number * 10
+    my_number = (my_number - ost) / 10
```
we have // and it would be simpler

belovmd commented on src/Homework2/task8.py:
```diff
@@ -0,0 +1,108 @@
+# 5 kyu task in www.codewars.com
+
+
+def make_readable(seconds):
```
too complex

belovmd commented on src/Homework2/task8.py:
```diff
@@ -0,0 +1,108 @@
+The tests contain some very huge arrays, so think about performance."""
+
+
+def find_uniq(arr):
```
too many magic numbers

belovmd reviewed (changes requested):
- missed tasks descriptions
- folder name contradicts task description

Talion90 committed:
Provide review remarks

add tasks descriptions,
change folder name contradicts task description,
task2: add functions for all "symbols"
task5: change solution to "a, b = b, b + a"
task6: using "//"
task8: make other solutions

Talion90 committed:
Provide review remarks

add tasks descriptions,
change folder name contradicts task description,
task2: add functions for all "symbols"
task5: change solution to "a, b = b, b + a"
task6: using "//"
task8: make other solutions

belovmd commented on src/homework2/task2.py:
```diff
@@ -0,0 +1,18 @@
+
+def find_max_word(str_):
+    formatted_string = "".join(letter for letter in str_
+                               if letter.isalpha() or letter.isspace() or letter.isdigit())
```
string.punctuation

belovmd reviewed (approved):
9
